index.js

```diff
@@ -399,6 +399,36 @@
     function(x) { return $.FiniteNumber.test(x) && Number(x) !== 0; }
   );
 
+  //  Integer :: Type
+  $.Integer = $.NullaryType(
+    'sanctuary-def/Integer',
+    function(x) {
+      var max = Math.pow(2, 53) - 1;
```

Comment:
I'd like to define `MAX_SAFE_INTEGER` and `MIN_SAFE_INTEGER` "constants" as in [Sanctuary][1]. These definitions should live near the top of the file:

```diff
   'use strict';

   var $ = {};

+  var MAX_SAFE_INTEGER = Math.pow(2, 53) - 1;
+  var MIN_SAFE_INTEGER = -MAX_SAFE_INTEGER;
+
   var LEFT_SINGLE_QUOTATION_MARK = '\u2018';
   var RIGHT_SINGLE_QUOTATION_MARK = '\u2019';
```


[1]: https://github.com/plaid/sanctuary/blob/v0.7.1/index.js#L121-L122